app/models/enrollment.rb

```diff
@@ -6,6 +6,10 @@ class Enrollment < ActiveRecord::Base
   has_one :certificate
 
   validates :honor_code, acceptance: true
+  validates :student_id, uniqueness: {
+                           scope: :course_id,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.